staff/francis-hrom/strings/count-words.test.js

```diff
@@ -0,0 +1,7 @@
+console.assert(countWords("    ")=='0','This is wrong result! -> ',countWords("    "));
```

Comment:
check should be done against numbers, not strings, so change the comparisons to work with numbers (e.g. `0` instead of `'0'` and so on)